buzzmobile/constants.yaml

```diff
@@ -44,7 +44,7 @@ wheel_separation: 1.3716 # distance between left and right wheels in meters
 max_steering_angle: 0.262 # radians (=15 degrees)
 
 # Max speed of buzzmobile in m/s
-max_speed: 2 #TODO this is a murder machine
+max_speed: 1 #TODO: measure the max speed.
```

Comment:
I can sort the issues, i just want us to track TODOs as issues. It's fine for the "TODO" to exist in code too.